src/utils/sortBy.js

```diff
@@ -0,0 +1,36 @@
+export const sortOldestFirst = (arr) => {
+  arr.sort((a, b) => {
+    return a.data.dateCreated - b.data.dateCreated;
+  });
+  return arr;
+};
+
+export const sortLatestFirst = (arr) => {
+  arr.sort((a, b) => {
+    return b.data.dateCreated - a.data.dateCreated;
+  });
+  return arr;
+};
+
+export const sortByTrending = (arr) => {
+  arr.sort((a, b) => {
+    return b.data.likes - a.data.likes;
+  });
+  return arr;
+};
+
+export const sortPosts = (arr, sortOrder) => {
+  switch (sortOrder) {
+    case "OLDEST":
```

Comment:
You can make these cases as Constants to avoid spell mistakes

Reply:
thanks for the suggestion